frontend/src/components/Common/Help.js

```diff
@@ -10,14 +11,17 @@ function Help(props) {
   return props.trigger ? (
     <div className='Help'>
       <div className='Help-inner'>
-        <button
-          className='SmallBtn'
-          id='CloseButton'
-          onClick={() => props.setTrigger(false)}
-        >
-          close
-        </button>
-        {props.children}
+        <Alert variant='info'>
```

Comment:
Nice work here Anmol. Not a change request, just making a small note there that quite a few commonly used button might be changed as well to what was done here.
The map button, and help button is used throughout multiple pages.